style.scss

```diff
@@ -0,0 +1,1183 @@
+            margin: 22px auto;
+          }
+
+          div {
+            p {
+              @include fontProp(14px, 24px, #191919);
+              font-weight: 400;
+
+              &:first-child {
+                margin-bottom: 20px;
+              }
+            }
+
+            div {
+              margin: 11px 0 24px 0;
+              display: flex;
+              align-items: center;
+
+              img {
+                padding-right: 20.5px;
+              }
+
+              p {
+                @include fontProp(12px, 24px, #7f7f7f);
+              }
+            }
+          }
+        }
+
+      }
+
+      .accordionHeader {
+        cursor: pointer;
+        display: flex;
+        align-items: baseline;
+        padding-right: 10px;
+        //vertical-align: middle;
+        h3 {
+          @include fontProp(16px, 24px, #191919);
+          font-weight: 400;
+          position: relative;
+          bottom: 3px;
+          flex-grow: 2;
+        }
+
+
+        span {
+          &:first-child {
+            font-family: 'Actor', sans-serif;
+            @include fontProp(40px, 48px, #E5E5E5);
+            font-weight: 400;
+            //text-transform: uppercase;
+            letter-spacing: 2px;
+            padding: 0 18px 0 12px;
+
+          }
+
+          &:last-child {
+
+            position: relative;
+            display: inline-block;
+            width: 14px;
+            height: 14px;
+            border: 1.5px solid #22BF95;
+            border-radius: 50%;
+            box-shadow: 0 3px 6px rgba(127, 127, 127, 0.3);
+
+            &::before, &::after {
+              content: "";
+              width: 7px;
+              height: 1.5px;
+              background-color: #22BF95;
+              position: absolute;
+              left: 2px;
+              top: 4.75px;
+              transition: transform 0.3s ease 0s;
+            }
+
+            &::after {
+              transform: rotate(-90deg);
+            }
+
+          }
+        }
+      }
+
+      .accordionBody {
+        display: none;
+
+
+      }
+    }
+  }
+}
+
+// -----GLOSSARY---------
+.glossary {
+  padding: 24px 0;
+  font-family: "Aclonica", sans-serif;
+
+  .title {
+    padding: 0 20px;
+    margin-bottom: 24px;
+
+    hr {
+      margin-bottom: 10px;
+      border: none;
+      height: 1.5px;
+      background-color: #B2B2B2;
+    }
+
+    img {
+      width: 114px;
+      height: 18px;
+    }
+  }
+
+  .slider {
+    background: linear-gradient(45deg, #E1E1E1, #FFFFFF);
+    padding: 24px 20px;
+
+    .list {
+      position: relative;
+      background: #FFFFFF;
+      box-shadow: 0 12px 24px #E5E5E5;
+      border-radius: 5px;
+      @include fontProp(16px, 32px, #7F7F7F);
+
+      ul {
+        padding: 39px 0 41px 62px;
+
+        li {
+          list-style-type: none;
+          cursor: pointer;
+
+          &#active {
+            position: relative;
+
+            &::before {
+              content: "";
+              left: -54px;
```

Comment:
make this positioning other way

Reply:
What's wrong with this method? In this case, I don't know any other way.

Reply:
![image](https://user-images.githubusercontent.com/84564899/120492368-af79d100-c3c2-11eb-9f4e-38af21f4293d.png)
this way you will have the correct position with any width of line